internal/generators/transformers/random_int64.go

```diff
@@ -18,9 +14,25 @@ type Int64Limiter struct {
 	distance uint64
 }
 
-func NewInt64Limiter(minValue, maxValue int64) (*Int64Limiter, error) {
+// NewInt64Limiter - create limiter by int size. size is required for optional min and max.
+func NewInt64Limiter(minValue, maxValue int64, size int) (*Int64Limiter, error) {
+	if minValue == 0 || maxValue == 0 {
+		minThreshold, maxThreshold, err := int_utils.GetIntThresholds(size)
+		if err != nil {
+			return nil, err
+		}
+
+		if minValue == 0 {
+			minValue = minThreshold
```

Comment:
The `0` value might be received from parameters. Consider the way how to overcome this obstacle. For instance, we would receive the values by pointer and if there is a nil, then set default value thresholds. 

`Parameterizer` interface provides the next method

```go
IsEmpty() (bool, error)
```

You can check that the value is empty if needed